spec/tags/whatsapp.md

```diff
@@ -34,3 +34,26 @@ When you receive a message from one contact, the sender and recipient is inverte
 * Sender: is the phone number of contact
 
 In API the sender is the field `from` and the receiver is the field `to` of message object.
+
+
+## Type of Messages
+
+* **Template Message**
+
+**What's the template message?**
+
+Message Templates are message formats for common reusable messages a business may want to send. Businesses must use Message Templates for sending notifications to customers.
```

Comment:
We should stick with the term "template message" since it is how they are called on our API and never use "message template".

Reply:
Even better if we are able to use the term content instead, since in the API Template is a type of Content.